src/components/About/Blog/index.js

```diff
@@ -280,6 +281,41 @@ class Blog extends Component {
     this.scrollToTop();
   };
 
+  handleClick = index => e => {
+    this.expandedElements.push(index);
+    this.setState({
+      expandedElements: this.expandedElements,
+    });
+  };
+
+  handleClickRemove = index => e => {
+    this.expandedElements.splice(this.expandedElements.indexOf(index), 1);
+    this.setState({
+      expandedElements: this.expandedElements,
+    });
+  };
+
+  handleSearch = value => {
+    let displayCount = 0;
+    const { actions } = this.props;
+    var list = document.getElementsByClassName('section_blog');
```

Comment:
We should not use these APIs like `getElementsByClassName`. Please refrain from using these. You can directly use this.state.posts